Libraries/Components/Touchable/TouchableOpacity.js

```diff
@@ -111,7 +115,7 @@ var TouchableOpacity = React.createClass({
 
   touchableHandlePress: function(e: Event) {
     this.clearTimeout(this._hideTimeout);
-    this._opacityActive();
+    this._opacityActive(150);
```

Comment:
I'm not certain what the best value is here, as I'm not confident what setting the element to active here is accomplishing, but this keeps the behaviour the same as previously.